openedx/adg/lms/applications/migrations/0003_user_application_update_business_line.py

```diff
@@ -23,7 +23,7 @@ class Migration(migrations.Migration):
                 ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, editable=False, verbose_name='created')),
                 ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, editable=False, verbose_name='modified')),
                 ('title', models.CharField(max_length=255, unique=True, verbose_name='Title')),
-                ('logo', models.ImageField(null=True, upload_to='business-lines/logos/', validators=[django.core.validators.FileExtensionValidator(('png', 'jpg', 'svg')), openedx.adg.lms.applications.helpers.validate_logo_size], verbose_name='Logo')),
+                ('logo', models.ImageField(help_text='Accepted extensions: .png, .jpg, .svg', null=True, upload_to='business-lines/logos/', validators=[django.core.validators.FileExtensionValidator(('png', 'jpg', 'svg')), openedx.adg.lms.applications.helpers.validate_logo_size], verbose_name='Logo')),
```

Comment:
This migration will not execute since it is already run on dev, do we intend to un-apply all migrations first on dev?